### PR TITLE
Fix some typos in issue-triage guide

### PR DESCRIPTION
**Pull request by abhiraut**
Fixing some spelling mistakes in the issue-triage guide.

### CODE REVIEW

File: contributors/guide/issue-triage.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ Ideally the SIG lead should have a SIG member that is a first point
 of contact for SIG new issues. If an issue has a SIG label assigned and no
 action is taken by SIG in 30 days (e.g. no comment was added by SIG or no
 discussion was initiated) then gently poke SIG about this pending issue.
-Also consider attending one of the SIG meetings and brig up issue, if you feel
+Also consider attending one of the SIG meetings and bring up issue, if you feel
```
Comment (justaugustus) [inline contributors/guide/issue-triage.md]:
```suggestion
Also, consider attending one of the SIG meetings and bringing up the issue, if you feel
```

**Comment by guineveresaenger** [comment]
With regards to content:
/approve

File: contributors/guide/issue-triage.md (hunk 1 above)
Comment (abhiraut) [reply contributors/guide/issue-triage.md]: thanks for your suggestion. Done

**Comment by mrbobbytables** [comment]
Nope, 1 commit in these situations is preferred. 😄 
Thanks!
/lgtm

**Pull request merged**